packages/@aws-cdk/aws-rds/test/instance.test.ts

```diff
@@ -1337,6 +1337,33 @@ describe('instance', () => {
     });
   });
 
+  test('can initialize copyTagsToSnapshot to true', () => {
+    new rds.DatabaseInstance(stack, 'Instance', {
+      engine: rds.DatabaseInstanceEngine.mysql({
+        version: rds.MysqlEngineVersion.VER_8_0_19,
+      }),
+      vpc,
+      copyTagsToSnapshot: true,
```

Comment:
Surprised that this works...? `DatabaseInstance` takes in `DatabaseInstanceProps`, which do not extend `InstanceProps`...?